src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -155,13 +155,13 @@ <h4 class="modal-title">Register Workflow</h4>
                   <div class="input-group">
                     <div class="input-group-btn">
                       <button type="button" class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown">
-                        {{hostedWorkflow?.descriptorType}}
+                        {{hostedWorkflow?.descriptorType.toUpperCase()}}
                         <span class="caret"></span>
                       </button>
                       <ul class="dropdown-menu">
                         <li *ngFor="let descriptorType of (descriptorLanguages$ | async)">
                           <a class="dropdown-item" (click)="hostedWorkflow.descriptorType = descriptorType.toLowerCase()">
-                            {{descriptorType | lowercase}}
+                            {{descriptorType | uppercase}}
```

Comment:
I haven't actually run the code, but won't this will always uppercase `NEXTFLOW`? Presumably we'd want it to be `Nextflow`?

Reply:
Yes.  But as it stands, there's no quick and clean way of fixing that.  I actually have a google docs sheet that will be presented to iron out these issues.

Reply:
I think I'd probably prefer `Nextflow` for display purposes, but will discuss in meeting